db/schema.rb

```diff
@@ -1006,6 +1006,15 @@
     t.index ["topic_id"], name: "index_statistics_announcements_on_topic_id"
   end
 
+  create_table "sync_check_results", id: :integer, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
+    t.string "check_class"
+    t.integer "item_id"
+    t.text "failures"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["check_class", "item_id"], name: "index_sync_check_results_on_check_class_and_item_id", unique: true
+  end
+
```

Comment:
This looks like a spurious change. I'd suggest removing it, unless you're sure this change should be made to the schema.rb file.

Reply:
Ah yeah for some reason there was another migration that ran, I've removed it now